### PR TITLE
Recipe generator example (aka octofridge)

### PR DESCRIPTION
**Pull request by tmoreau89**
Thanks to @adstraw and @dpk-dmo who added the demo and documentation

### CODE REVIEW

**Review by AI-Bassem — COMMENTED** [review]
LGTM

**Review by AI-Bassem — APPROVED** [review]

File: recipe_generator/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# How to build a food recipe generator with OctoAI SDK and Llama2 LLM
+
+This is a Python command line interface (CLI) application that takes as inputs a list of ingredients from the user and responds with a recipe for a dish that can be prepared from the provided list of ingredients. It demonstrates the ease with which large language models (LLM) powered logic and reasoning can be integrated into applications to deliver engaging experiences to end users. The application uses the OctoAI Python SDK and the Llama 2 13B Chat model on the OctoAI Text Gen Solution.
```
Comment (chailandau) [inline recipe_generator/README.md:3]:
```suggestion
This is a Python command line interface (CLI) application that takes a list of ingredients from the user as inputs and responds with a recipe for a dish that can be prepared from the provided list of ingredients. 

It demonstrates the ease with which large language models (LLM) powered logic and reasoning can be integrated into applications to deliver engaging experiences to end users. 

The application uses the OctoAI Python SDK and the Llama 2 13B Chat model on the OctoAI Text Gen Solution.
```

File: recipe_generator/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Features
+
+This example doesn't rely on Langchain and uses the OctoAI SDK directly which is OpenAI API compatible! Here are the features that you'll be using to build your recipe generator.
```
Comment (chailandau) [inline recipe_generator/README.md:7]:
```suggestion
This example doesn't rely on Langchain and uses the OctoAI SDK directly, which is OpenAI API compatible! Here are the features that you'll be using to build your recipe generator:
```

File: recipe_generator/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+* OctoAI Python sdk (octoai-sdk)
+* Text generation using Llama 2, using the OpenAI chat completions API
```
Comment (chailandau) [inline recipe_generator/README.md:10]:
```suggestion
* Text generation with Llama 2, using the OpenAI chat completions API
```

File: recipe_generator/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+```bash
+python3 -m pip install --upgrade pip
+python3 -m pip install -r requirements.txt
+```
+
+### Environment setup
+
+To run our example app, there are two simple steps to take:
+
+- Get an OctoAI API token by following [these instructions](https://docs.octoai.cloud/docs/how-to-create-an-octoai-access-token).
```
Comment (chailandau) [inline recipe_generator/README.md:35]: https://octo.ai/docs/getting-started/how-to-create-octoai-api-token/

Update url to the new docs

Comment (tmoreau89) [reply recipe_generator/README.md]: Thank you, updated!

**Comment by tmoreau89** [comment]
Made some updates to other READMEs to make them consistent.

**Comment by tmoreau89** [comment]
Thank you @chailandau @AI-Bassem @dpk-dmo @adstraw - the PR has been merged!

**Pull request merged**